python/dgl/backend/__init__.py

```diff
@@ -7,7 +7,8 @@
 
 from . import backend
 from .set_default_backend import set_default_backend
-
+from .pytorch.sparse import gspmm_hetero
+from .pytorch.sparse import gsddmm_hetero
```

Comment:
we should import any function in pytorch backend here.

Reply:
Putting it inside the pytorch folder doesn't work. Where should I put it?